app/assets/stylesheets/common.sass

```diff
@@ -3,3 +3,6 @@
 
 .avatar-mini
   width: 40px
+
+.inline-indent
```

Comment:
I think that's a bad name for such class, you won't be able to recognise the purpose from view code.